src/explore-education-statistics-admin/src/pages/release/datablocks/components/DataBlockPageTabs.tsx

```diff
@@ -171,14 +171,18 @@ const DataBlockPageTabs = ({
         const majorAxis = draft[0]?.axes?.major;
         const legend = draft[0]?.legend;
 
-        // Remove data sets that are no longer applicable to a given table's subject meta.
+        // If old chart title is the same as the old table title, then the chart title is defaulting to
```

Comment:
Do we need a migration to look at existing Chart titles versus their table titles and unset them if they're the same.

We did this for Methodologies adopting their owning Publications' titles.

Reply:
We shouldn't need to. As far as the frontend is concerned, when getting a chart's title, it gets the same response when a chart title is null in the DB and when the chart title is the same as the table heading in the DB. When the frontend changes a data blocks details, it doesn't set a chart title if the chart title is the same.

So older charts that have a table heading and chart title that are the same both set in the DB will get coerced into a table heading and null chart title in the DB whenever someone next updates that data blocks in the frontend.